source/codegen/generate_service.py

```diff
@@ -23,10 +23,6 @@ def generate_service_file(metadata, template_file_name, generated_file_suffix, g
 
 def mutate_metadata(metadata: dict):
     config = metadata["config"]
-    for custom_type in config.get("custom_types", []):
```

Comment:
Custom types had a similar override that was implemented slightly differently. Instead of relying on this patch, They now use the above helpers.